auto.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+var shim = require('./shim');
+module.exports = shim();
```

Comment:
there's not really a need to export anything from this file; it'd be better if it just does `require('./shim')()`